python/ray/ml/predictors/integrations/torch/torch_predictor.py

```diff
@@ -0,0 +1,67 @@
+from typing import Optional, Union, List
+
+import torch
+
+from ray.ml.predictor import Predictor, DataBatchType
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+
+
+class TorchPredictor(Predictor):
+    """A predictor for PyTorch models.
```

Comment:
Can you add an example of using TorchPredictor here with a simple model and data batch?

Reply:
(show both numpy and pandas inputs)

Reply:
Added example!